build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -106,12 +116,17 @@ def log(string, *args, indent=0, **kwargs):
         file=sys.stderr)
 
 
-def convert_directories(directories, write_files, allow_partial_conversion):
+def convert_directories(directories, write_files, allow_partial_conversion,
+                        verbosity):
   failure_dirs = []
   skip_count = 0
   success_count = 0
```

Comment:
nit: would be nice to note how many "successfully updated" files were actually modified and differentiate between the script being a no-op and actually editing some files on disk (see the suggested output on https://github.com/google/iree/issues/4282). We'd have to actually inspect the files to be able to know that though.

Reply:
https://github.com/google/iree/pull/4951/files